Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -688,26 +685,19 @@ bool AccessibilityNodeObject::computeAccessibilityIsIgnored() const
 
 bool AccessibilityNodeObject::canvasHasFallbackContent() const
 {
-    Node* node = this->node();
-    if (!is<HTMLCanvasElement>(node))
-        return false;
-    HTMLCanvasElement& canvasElement = downcast<HTMLCanvasElement>(*node);
+    RefPtr canvasElement = dynamicDowncast<HTMLCanvasElement>(node());
```

Comment:
The previous code was null-checking the node so passing `node()` here is correct.